packages/schemas/src/metadata/field/get-leaderboard-metadata.ts

```diff
@@ -65,6 +65,7 @@ export const getLeaderboardMetadata = (
 
   return {
     enabled: true,
+    historical: leaderboardOptions?.historical ?? true,
```

Comment:
Don't default this to true. It causes issues woth carrying down the property.

Reply:
It only gets default to true when the field is a leaderboard, so this true value will never even carry down.